docs/utils/iers.rst

```diff
@@ -0,0 +1,165 @@
+.. _utils:
```

Comment:
@pllim - thanks!  I got sidetracked about the error it was reporting and usually expect Sphinx messages to be misleading or useless.  In this case though I should have been able to see it.